src/Polly.Core/ResilienceStrategyExtensions.Async.ValueTask.cs

```diff
@@ -0,0 +1,81 @@
+using System.Threading;
+using Polly;
+
+namespace Polly;
+
+/// <summary>
+/// The extensions for <see cref="IResilienceStrategy"/>. These extensions are used to execute the user callbacks in various execution modes.
+/// For example, you can use the same <see cref="IResilienceStrategy"/> instance to execute a synchronous callback,
+/// an asynchronous callback, or a callback that returns a void result.
+/// </summary>
+public static partial class ResilienceStrategyExtensions
+{
+    /// <summary>
+    /// Executes the <paramref name="execution"/> callback.
+    /// </summary>
+    /// <typeparam name="TState">The type of s associated with the execution.</typeparam>
```

Comment:
`state` rather than `s`?

Reply:
Same comments in the other places this comes up rather than adding too much comment noise.

Reply:
Thanks for spotting this. Fixed.